.gitignore

```diff
@@ -12,3 +12,6 @@ env
 # notebooks
 .ipynb_checkpoints
 *.ipynb
+
+# venv
+venv
```

Comment:
pull request should be atomic, if you wanna update .gitignore do it, but in a separate pull request